src/platform/glfw.rs

```diff
@@ -24,8 +24,7 @@ pub fn init(
     glfw.window_hint(glfw::WindowHint::Focused(true));
     glfw.window_hint(glfw::WindowHint::RefreshRate(None));
     glfw.window_hint(glfw::WindowHint::ScaleToMonitor(true));
-    glfw.window_hint(glfw::WindowHint::DoubleBuffer(false));
-
+    glfw.window_hint(glfw::WindowHint::DoubleBuffer(true));
```

Comment:
This should be kept as `false`, like in your example!

Reply:
I apologize again, as corrected in a comment on the tearing issue the example was wrong, the text proceeding the example described the solution correctly i.e. having double buffer enabled while setting the swap interval to 0/None to disable Vsynd